NatanimA/Javascript-capstone-project
# External - Reviews

**Please Note that we were two and we have done up to the comments feature**

> Presentation Link https://drive.google.com/file/d/1l3tjUHZ3Pg7TJk6XCkybGSyrFQJyvutS/view

> Live link https://natanima.github.io/Javascript-capstone-project/

- [ ] Implemented The homepage Movies Fetching
- [ ] Implemented The homepage Item counter
- [ ] Implemented The homepage Likes
- [ ] Implemented Likes counter upon each like
- [ ] Implemented Comment section Pop up
- [ ] Implemented Comment section pop up comments fetching by given ID
- [ ] Implemented Add comment feature to each movies
- [ ] Implemented Links leading to official web page
- [ ] Implemented Mobile version
- [ ] Implemented All the other requirements that was given
- [ ] Implemented Tests to check Item counter
- [ ] Implemented Tests to check Comment counter
- [ ] Deployed the page using gh-pages

- To run it follow this steps

```
git clone https://github.com/NatanimA/Javascript-capstone-project.git
```

```
npm install
```

```
npm run build
```

Lastly

```
npm start
```

😉 

To Test it follow this steps

Please make sure to download the dependencies first
```
npm install --save-dev jest
```
JS-DOM is needed as well
```
npm install -D jest-environment-jsdom
```
Lastly

```
npm test
```

😉
